src/sds_data_model/_table.py

```diff
@@ -0,0 +1,26 @@
+from pandas import to_numeric
+   
+# Updates datatypes to smaller memory allocation, except dtype int8 which is not accepted by rasterio package, so converted to int16
+def _update_datatypes(
+    df 
+):    
```

Comment:
Move comment to docstring and add type hints. 

Your docstring should follow the [Google docstring style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) and should include a single line description, a list of arguments and types, a return value and type, and an example.

You can use the VS Code  [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) plugin to generate most of this for you.

```suggestion
from pandas import DataFrame, to_numeric

    
 def _update_datatypes(
    df: DataFrame,
) -> DataFrame:    
"""Updates datatypes to smaller memory allocation, except dtype int8 which is not accepted by rasterio package, so converted to int16."""
```